app/views/teams/_form.html.slim

```diff
@@ -69,8 +69,8 @@
               = s.link_to_remove 'Remove', class: 'btn btn-default'
       .form-group.form-btn-group
         = f.link_to_add 'Add another source', :sources, class: 'btn btn-primary col-sm-offset-3'
-  -if can?(:update_conference_preferences, @team)
-  = render 'conference_preference_fields', f: f
+  - if can?(:update_conference_preferences, @team) && @team.accepted?
```

Comment:
As far as I remember the ability `update_conference_preferences` was introduced specifically for this view / this context. So I would say if we change the restrictions, this should be done inside this specific ability itself, don't you think?
It would also have the benefit of having the view less cluttered with conditions 😉 

So what do you think about adapting the resembling ability and its tests instead?